devel/ci/run_tests.sh

```diff
@@ -59,6 +59,7 @@ mv development.ini.example development.ini
 
 sudo /usr/bin/python setup.py develop
 
-/usr/bin/python setup.py nosetests
+/usr/bin/pytest
+/usr/bin/tox -e flake8,pydocstyle
```

Comment:
It might be nice to swap the order of these two, since the style tests run more quickly - we could give the user faster feedback on style errors. Just a suggestion.